### PR TITLE
Make Extended Great Again!

### PR DESCRIPTION
**Pull request by ghost**
Refer to issue #1708 .

#### Intent of my Pull Request

- [x] Extended is in rotation with a 6% chance of occurring (other gamemode chances reduced- see code for more details)
- [x] Implement station objectives.

##### Changelog

:cl:
tweak: It might just be possible that hostilities do not occur for an entire shift (extended in rotation).
rscadd: The crew of Space Station 13 is now expected to partake in station-wide goals directed by Nanotrasen.
/:cl:

### CODE REVIEW

**Comment by drderp3635** [comment]
it would be neat if you could add the possibilty of the magic events with a low roll chance

**Comment by Dagdammit** [comment]
2%? Quick review indicates yogs had 86 rounds in the last 5 days, so that's what, average of 1 extended round every 3 IRL days? I'd be fine with something a few percentiles higher.

**Comment by drderp3635** [comment]
Should probably raise it to 4 or 5. Also you should note on here that you are reducing the chance of specific modes

**Comment by matskuman5** [comment]
> Implement station objectives

This is worth it's own PR, I've been wishing for this ever since Goon added them

**Comment by ghost** [comment]
@matskuman5 Station objectives are a thing on /tg/ that is basically building a few things and then you get some ridiculous machine or wahtever

**Comment by Dagdammit** [comment]
https://github.com/tgstation/tgstation/blob/master/code/game/gamemodes/extended/extended.dm
https://github.com/tgstation/tgstation/tree/master/code/modules/station_goals

Interesting stuff, I'll see if I can track down info on the goon stuff too.

**Comment by ghost** [comment]
The /tg/ station goals seem boring. What has Goon done with it?

**Comment by Dagdammit** [comment]
I poked around in the 2016 release and couldn't find anything?

**Comment by ghost** [comment]
I have no idea what's wrong. It compiles fine for me!

File: code/__HELPERS/icons.dm [hunk 1]
```diff
@@ -248,6 +248,27 @@ world
 	I.Blend(icon, ICON_SUBTRACT)
 	Blend(I, ICON_SUBTRACT)
 
+/atom/proc/cut_overlays()
```
Comment (Super3222) [inline code/__HELPERS/icons.dm]: this was already defined is the problem. I recently ported this from /tg/ even. You're going to have to pull in those changes, and fix the conflicts

File: config/game_options.txt [hunk 2]
```diff
@@ -82,18 +82,18 @@ ALERT_DELTA Destruction of the station is imminent. All crew are instructed to o
 
 PROBABILITY TRAITOR         7
 PROBABILITY TRAITORCHAN     7
-PROBABILITY DOUBLE_AGENTS     7
-PROBABILITY NUCLEAR         10
+PROBABILITY DOUBLE_AGENTS     6
```
Comment (Super3222) [inline config/game_options.txt:85]: You'll need the council to update their files manually for these changes.

**Comment by BOBsHERE** [comment]
6% chance is so high. I know its really not, but extended rounds on TG are horribly boring and griefy. On yogstation it will be much, much worse. Maybe make it only fire if an admin is on (not a mod, an admin that can spawn shit) so that they can at least get centcom involved in making the round progress.

**Comment by Dagdammit** [comment]
Is that actually doable? Could be a smart move if admin intends to stick
around- like it could ask the admin if they're up for it?

**Comment by ghost** [comment]
It's done!

**Comment by matskuman5** [comment]
So uh what are the station objectives like?

**Comment by Cruix** [comment]
Not updating the gamemode chances yet, just going to test the station goals.

**Pull request merged**